lightningd/opening/opening.c

```diff
@@ -658,23 +644,25 @@ static u8 *fundee_channel(struct state *state,
 		      &state->our_secrets.funding_privkey,
 		      our_funding_pubkey, &sig);
 
+	/* We don't send this ourselves: master does, because it needs to save
+	 * state to disk before doing so. */
 	msg = towire_funding_signed(state, &channel_id, &sig);
-	if (!sync_crypto_write(&state->cs, PEER_FD, msg))
-		peer_failed(PEER_FD, &state->cs, NULL, WIRE_OPENING_PEER_WRITE_FAILED,
-			      "Writing funding_signed");
-
-	return towire_opening_fundee_finish_reply(state,
-						  state->funding_txout,
-						  state->remoteconf,
-						  &theirsig,
-						  &state->cs,
-						  &their_funding_pubkey,
-						  &theirs.revocation,
-						  &theirs.payment,
-						  &theirs.delayed_payment,
-						  &state->next_per_commit[REMOTE],
-						  state->funding_satoshis,
-						  state->push_msat);
+	encmsg = cryptomsg_encrypt_msg(state, &state->cs, msg);
```

Comment:
Just in case there's confusion, this was true in one patch (saved master doing the peer crypto), but then when I moved it to the connect daemon, this code here was cleaned up to hand it in cleartext as expected.